src/App.jsx

```diff
@@ -1,20 +1,20 @@
 import { useState } from 'react'
 import "./styles.css"
-
+import DarkModeOutlinedIcon from '@mui/icons-material/DarkModeOutlined'
 
 function App() {
 
   return (
     <>
-      <div>
+      <div className='bar'>
         <h3 className='name'>Elias</h3>
-        <img />
+        <DarkModeOutlinedIcon className='dark-mode' fontSize='large'/>
       </div>
       <div>
         <h1 className='ocupation'>Frontend</h1>
         <h1 className='ocupation'>Developer</h1>
         <h4 className='ocupation-message'>I like to craft solid and scalable frontend projects with great user experiences.</h4>
```

Comment:
```suggestion
      <Box>
        <Typography variant="h1" className='ocupation'>Frontend</Typography>
        <Typography variant="h1"  className='ocupation'>Developer</Typography>
        <Typography variant="h4"  className='ocupation-message'>I like to craft solid and scalable frontend projects with great user experiences.</Typography >
```

MUI makes it easy to add divs and h1s by using their built in components. 
https://mui.com/material-ui/react-typography/
https://mui.com/material-ui/react-box/

You don't need to use these. But I would recommend giving them a try and see if you can get it to work. Mae sure you import the components. You can find tons of examples in the documentation I linked above